Sonarr not finding 7z files from usenet

there we go - same folder with debug enabled

> [ERROR] 2025/03/25 12:55:09.688025 handlers.go:216: Extraction Failed: Jimmy.Kimmel.2025.03.13.Ellen.Pompeo.720p.WEB.h264-EDITH: no compressed files found

Origin user: helmi
Origin message: [Discord Server Link](https://discord.com/channels/184509568867368963/781625367391371274/1354124185768693771)
Reporting user: twitchcaptain
Reporting message: [Discord Server Link](https://discord.com/channels/184509568867368963/781625367391371274/1361431419951382649)

--- a/pkg/unpackerr/handlers.go
+++ b/pkg/unpackerr/handlers.go
@@ -127,7 +127,7 @@ func (u *Unpackerr) extractCompletedDownload(name string, now time.Time, item *E
 		Filter: xtractr.Filter{
 			Path: item.Path,
 			ExcludeSuffix: xtractr.AllExcept([]string{
-				".rar", ".r00", ".zip", ".7z", ".gz", ".tgz", ".tar", ".tar.gz", ".bz2", ".tbz2",
+				".rar", ".r00", ".zip", ".7z", ".7z.001", ".gz", ".tgz", ".tar", ".tar.gz", ".bz2", ".tbz2",
 			}),
 		},
 		TempFolder: false,

